fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -1286,9 +1286,9 @@ private PutLoansLoanIdRequest() {}
         @Schema(example = "true")
         public Boolean createStandingInstructionAtDisbursement;
         @Schema(example = "1")
-        public Integer repaymentFrequencyNthDayType;
+        public Long repaymentFrequencyNthDayType;
```

Comment:
This looks wrong to me based on LoanScheduleAssembler line 196. It is the ordinal of an enum.